scripts/error-codes/formatProdErrorMessage.js

```diff
@@ -12,9 +12,17 @@
 // template literal strings. The messages will be replaced with error codes
 // during build.
 
-function formatProdErrorMessage(code) {
+function formatProdErrorMessage(code, adj, noon) {
```

Comment:
```suggestion
function formatProdErrorMessage(code, adj, noun) {
```

`noun` is how it's called inside `transform-error-messages.js`